aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -120,6 +120,7 @@ public S3FileIO(SerializableSupplier<S3Client> s3) {
    */
   public S3FileIO(SerializableSupplier<S3Client> s3, S3FileIOProperties s3FileIOProperties) {
     this.s3 = s3;
+    this.s3FileIOProperties = s3FileIOProperties;
```

Comment:
Thanks for the fix. Down the line AWS integration tests being run during CI should catch issues like this.